### ISSUE TITLE
How to Debug search-extensibility-library project

### ISSUE THREAD

**Issue by pravin080317** [microsoft-search/pnp-modern-search#102]
I have tried gulp serve and debugged using local host/ hosted workbench but break point is not getting resolved. Kinldy assist me on debugging the library project

**Comment by FranckyC** [comment]
Hi @pravin080317 To debug the library, you must: 
- Package and deploy a production version of the PnP modern search solution either using the `--ship` option or use the one from the release package.
- Run `gulp serve --nobrowser` from the extensibility solution using the hosted workbench (not local).
- Add the 'Search Results' Web Part to the page. It will load the extensibility dynamically and trigger your breakpoints.

In summary you can't debug both at once :(

**Comment by pravin080317** [comment]
FranckyC,

Thanks for the quick response. I have tried exactly the same thing before your comments and again tried after your comments.

Deployed the production ready PnP modern search solution using --ship.
Run gulp serve --nobrowser from the extensibility solution using the hosted workbench.
Added 'search box' and search results webpart to the page. 
But still the breakpoints are not triggered.

![spfxbreakpointissue](https://user-images.githubusercontent.com/35328578/74911289-514df800-53e2-11ea-938f-5b60b4268390.png)

**Comment by wobba** [comment]
@pravin080317 If so, I would consider this a SPFx problem (github.com/sharepoint/sp-dev-docs) and not modern search related. Can you start a new scratch project for extensibility and see if that works.

**Comment by pravin080317** [comment]
@wobba i have created scratch project for extensibility and found the same problem. raised the new issue regarding the same in the below forum.
https://github.com/SharePoint/sp-dev-docs/issues/5371

**Comment by FranckyC** [comment]
Hi @pravin080317 forgot to mention, you have also to package the extensibility solution using gulp package-solution (without `--ship`) and then upload to your app catalog (global or site). Then retry.

**Comment by pravin080317** [comment]
@FranckyC : Thanks without packaging --ship solved the issue. i can able to debug the extensibility solution now. 
I will be working on the custom suggestions to make a call to existing azure function and retrieve some of the data to be passed to search box.

Thanks for your kind assistance.

**Comment by FranckyC** [comment]
Coo
